packages/video_player/example/pubspec.yaml

```diff
@@ -8,7 +8,9 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-
+  flutter_driver:
```

Comment:
missing flutter version constraints for example app.

Reply:
Not needed here since it matches the Flutter SDK constraints of the plugin itself.